proxy/http.go

```diff
@@ -89,6 +89,22 @@ func DefaultHTTPResponseParserFactory(cfg HTTPResponseParserConfig) HTTPResponse
 	}
 }
 
+// NopHTTPResponseParserFactory is a HTTPResponseParserFactory implementation that just copies the
+// http response body into the proxy response IO
+func NopHTTPResponseParserFactory(_ HTTPResponseParserConfig) HTTPResponseParser {
+	return func(_ context.Context, resp *http.Response) (*Response, error) {
+		return &Response{
+			Data:       map[string]interface{}{},
+			IsComplete: true,
+			Io:         resp.Body,
```

Comment:
the response body should be wrapped with the `readCloserWrapper` defined at https://github.com/devopsfaith/krakend/blob/master/proxy/proxy.go#L32